ci-conda.Dockerfile

```diff
@@ -101,8 +123,8 @@ case "${LINUX_VER}" in
     rm -rf "/var/lib/apt/lists/*"
     ;;
   "rockylinux"*)
-    yum update -y
-    yum clean all
+    dnf update -y
```

Comment:
Aligning these on `dnf`, we have had a mixture of `yum` and `dnf` but it seems that `dnf` is the preferred / modern command to use.